TransportOperator/transportOperatorDT_class.f90

```diff
@@ -49,6 +49,9 @@ subroutine deltaTracking(self, p, tally, thisCycle, nextCycle)
     ! Get majornat XS inverse: 1/Sigma_majorant
     majorant_inv = ONE / self % xsData % getMajorantXS(p)
 
+    ! Should never happen! Prevents NaN distances
+    if (majorant_inv /= majorant_inv) call fatalError(Here, "Majorant is 0")
+
     DTLoop:do
       distance = -log( p% pRNG % get() ) * majorant_inv

```

Comment:
```suggestion
    DTLoop:do
      distance = -log( p% pRNG % get() ) * majorant_inv

      ! Should never happen! Prevents NaN distances    
      if (distance /= distance) call fatalError(Here, "Majorant is 0")
```

Is this better @Mikolaj-A-Kowalski ?

Reply:
Or we can check if the majorant is `0` so its inverse is `Infinity`? I don't know...  
I have a feeling that here we have a small chance to get a NaN. Really somebody would have to have `0` majorant and also sample `0` from RNG to get `0 * Inf` expression.  
```suggestion
    ! Should never happen! Prevents Inf distances
    if (abs(majorant_inv) > huge(majorant_inv)) call fatalError(Here, "Majorant is 0")

    DTLoop:do
      distance = -log( p% pRNG % get() ) * majorant_inv

```

Reply:
Yes sounds good!